src/renderer/api/endpoints/pods.api.ts

```diff
@@ -275,6 +275,9 @@ export class Pod extends WorkloadKubeObject {
     }[];
     hostIP: string;
     podIP: string;
+    podIPs?: {
+      ip: string
+    }[];
```

Comment:
there must be podIPs, we can remove the podIP

If there is only one ip, then it will have the following format:

```
podIPs: [{
  ip: "ip"
}]
```

Reply:
Hi @BlackHole1, yeah, we can. But the spec (https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L3638) still have this field. Maybe the spec still have this field for backward compatibility, PodIPs has been added in 1.16 kubernetes release (as I can see) https://github.com/kubernetes/kubernetes/blob/release-1.15/staging/src/k8s.io/api/core/v1/types.go

Reply:
If we want to follow the spec definition, then our `hostIP`, `podIP`, etc. also need to be changed to optional

Reply:
You're right.

Reply:
I would prefer if we didn't touch the fields at all. We can modify the methods so that `getPodIps()` returns the unique ip strings as an array if we want.